src/scripts/main.js

```diff
@@ -1 +1,25 @@
 'use strict';
+
+document.querySelector('DOMContentLoaded', () => {
+  const thumbsContainer = document.getElementById('thumbs');
+  const largeImg = document.getElementById('largeImg');
+
+  thumbsContainer.addEventListener('click', function (e) {
+    const target = e.target;
+
+    if (target && (target.tegName === 'IMG' || target.tagName === 'A')) {
+      e.preventDefault();
+
+      let newSrc;
+
+      if (target.tagName === 'A') {
+        newSrc = target.parentElement.href;
+      } else if (target.tagName === 'IMG') {
+        newSrc = target.parentElement.href;
```

Comment:
Similarly, using 'target.parentElement.href' assumes that the 'IMG' tag is wrapped directly by an 'A' tag. Ensure that this structure is correct or adjust the logic to find the correct 'A' tag.